code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -58,9 +58,9 @@
 
 /obj/item/weapon/melee/energy/suicide_act(mob/user)
 	var/tempgender = "[user.gender == MALE ? "he's" : user.gender == FEMALE ? "she's" : "they are"]"
-	if (active)
-		viewers(user) << pick("<span class='danger'>\The [user] is slitting \his stomach open with \the [src]! It looks like [tempgender] trying to commit seppuku.</span>", \
-		                      "<span class='danger'>\The [user] is falling on \the [src]! It looks like [tempgender] trying to commit suicide.</span>")
+	if(active)
+		user.visible_message(pick("<span class='danger'>\The [user] is slitting \his stomach open with \the [src]! It looks like [tempgender] trying to commit seppuku.</span>",
+			"<span class='danger'>\The [user] is falling on \the [src]! It looks like [tempgender] trying to commit suicide.</span>"), "You start commiting suicide.")

```

Comment:
That last one really shouldn't be there, the third input in a visible_message is a sound that is displayed to blind people, last I checked.
Also you need to end the span class.

Reply:
Span class is ended, on both quotes.

Reply:
There are only two inputs here. There's the 'pick' between the two suicide messages, and then there's the second, which is the message you see as the person.